content/organizations/managing-organization-settings/deleting-an-organization-account.md

```diff
@@ -26,7 +26,15 @@ shortTitle: Delete organization
 {% endtip %}
 
 {% endif %}
-Deleting your organization account removes all repositories, forks of private repositories, wikis, issues, pull requests, and project or organization pages. {% ifversion fpt or ghec %}Your billing will end and, after 90 days, the organization name becomes available for use on a new user or organization account.{% endif %}
+Deleting your organization account removes all repositories, forks of private repositories, wikis, issues, pull requests, and project or organization pages. {% ifversion fpt or ghec %}Your billing will end and, after 90 days, the organization name becomes available for use on a new user or organization account.
+
+{% tip %}
+
+**Tip**: To replace your organization, consider renaming it instead of deleting the existing one. Deleting an organization causes its name to be unavailable for 90 days, while renaming an organization allows you to create a new one with the same name immediately. For more information, see "[Renaming an organization](/organizations/managing-organization-settings/renaming-an-organization)."
```

Comment:
I suggest simplifying here because we're repeating some information. I think the below would be enough, what do you think?

```suggestion
**Tip**: If you rename an organization, you can create a new organization with the same name immediately. For more information, see "[AUTOTITLE](/organizations/managing-organization-settings/renaming-an-organization)."
```

Reply:
Ooh, I really like that. Shorter and to the point. It leaves the "why would you do that" question open, but I don't think that is a problem.

Also, nice to see this `AUTOTITLE` feature, I might have to implement it on my own projects 😛